modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -9,6 +8,10 @@
 	--n-accordion-item-title-justify-content-mobile: initial;
 	--n-accordion-title-icon-order: -1;
 	--n-accordion-title-icon-padding-inline: 0 15px;
+  	--n-accordion-item-title-space-between: 0px;
+  	--n-accordion-item-title-distance-from-content: 0px;
```

Comment:
@TzviRabinovitch can we shorten these class names to:
`--n-accordion-title-space-between:`
`--n-accordion-title-distance-from-content`